docroot/themes/custom/mass_theme/templates/field/mass_content_tableau_embed.html.twig

```diff
@@ -12,13 +12,24 @@
  * @see template_preprocess_link_formatter_link_separate()
  */
 #}
-
-<div class="ma_tableau_container" style="max-width:100%">
+{% include "@atoms/09-media/figure.twig" with {
+  "figure": {
+    "align": "right",
+    "dataviz": {
+      "id": randId,
+      "size": "medium",
+      "src": url
+    },
+    "title": "",
+    "caption": "This is an optional caption that can wrap to multiple lines."
+  }
+} %}
+{# <div class="ma_tableau_container" style="max-width:100%">
   <div id="{{ randId }}"></div>
 </div>
 <script type="text/javascript">
```

Comment:
Oh right - I think we need to keep the tableau embed code in Drupal, also.  My preference would be for Mayflower to allow us to inject raw HTML into the figure template, as I outlined in my response to massgov/mayflower#954

Reply:
In my proposed change on the other PR, here's how it would look on the Drupal side:
```twig
{% set tableau %}
<div class="ma_tableau_container" style="max-width:100%">
  <div id="{{ randId }}"></div>
</div>
<script type="text/javascript">
  window.addEventListener("DOMContentLoaded", function() {
    const url = "{{ url }}"
    new tableau.Viz(document.getElementById("{{ randId }}"), url);
  });
</script>
{% endset %}
{% include "@atoms/09-media/figure--dataviz.twig" with {
  "figure": {
    "align": "right",
    "content": tableau
    "title": "",
    "caption": "This is an optional caption that can wrap to multiple lines."
  }
} %}
```